client/me/security-2fa-initial-setup/index.jsx

```diff
@@ -35,10 +45,44 @@ class Security2faInitialSetup extends Component {
 					) }
 				</p>
 
+				<p>
+					<FormLabel>
+						<FormRadio
+							name="auth_method"
+							value="app-based"
+							defaultChecked={ true }
```

Comment:
Could be shortened to:

```suggestion
							defaultChecked
```